src/server/server.rs

```diff
@@ -28,15 +28,17 @@ use super::{Msg, SendCh, ConnData};
 use super::conn::{Conn, OnWriteComplete};
 use super::{Result, OnResponse};
 use util::HandyRwLock;
+use util::worker::Worker;
 use storage::Storage;
 use super::kv::StoreHandler;
-use super::coprocessor::EndPointHost;
+use super::coprocessor::{RequestTask, EndPointHost};
 use super::transport::RaftStoreRouter;
 use super::resolve::StoreAddrResolver;
 use raft::SnapshotStatus;
 
 const SERVER_TOKEN: Token = Token(1);
 const FIRST_CUSTOM_TOKEN: Token = Token(1024);
+const DEFAULT_COPROCESSOR_CONCURRENCY: usize = 50;
```

Comment:
Concurrency confuses me here, do we have 50 threads to do coprocessor? Seem batch_size here.